src/mirage_ci.ml

```diff
@@ -95,12 +95,13 @@ let main current_config github mode auth config
     | Some api -> Current_github.Api.webhook_secret api
     | None -> ""
   in
+  let get_job_ids ~owner:_ ~name:_ ~hash:_ = [] in
```

Comment:
FYI this can be used to request rebuilds via the GH Checks API which doesn't seem to be used here.
So I've just included a dummy no-op function.